### DISCUSSION TITLE
Parallelization Problem

### DISCUSSION THREAD

**Discussion by VinithSamson11** [SmileiPIC/Smilei#710] [Q&A]
Hello, 

I am trying to reduce the simulation time in SLURM. I could not able activate the OpenMP task Parallelization in smilei. Here is the output I am getting in "Initializing MPI" block,
```
Initializing MPI
 -------------------------------------------------------------------------------
	 MPI_THREAD_MULTIPLE enabled
	 Number of MPI processes: 96
	 Number of threads per MPI process : 1
	 OpenMP task parallelization not activated
 
	 Number of patches: 32 x 32
	 Number of cells in one patch: 32 x 8
	 Dynamic load balancing: every 20 iterations/
```
Please let me know how to activate the OpenMP task parallelization in-order to reduce simulation time. Thank You.

**Comment by mccoys** [comment]
Task parallelization is not available at the moment. Please disable it. 

Have you tried optimizing your simulation with mpi+openmp already?

**Reply by OwenSlevert** [reply]
How can the task parallelization then be disabled?

**Reply by mccoys** [reply]
Well actually it is already disabled in your case. I would try to use normal open mp threads (not tasks).
